mmv1/third_party/terraform/envvar/envvar_utils.go

```diff
@@ -111,6 +115,14 @@ func GetTestCredsFromEnv() string {
 	return transport_tpg.MultiEnvSearch(CredsEnvVars)
 }
 
+// Returns googleapis.com if there's no universe set.
+func GetTestUniverseDomainFromEnv(t *testing.T) string {
+	if transport_tpg.MultiEnvSearch(UniverseDomainEnvVars) == "" {
+		return "googleapis.com"
```

Comment:
We should SkipIfEnvNotSet similar to org-level tests or other similar external requirements! That'll mean that we clearly show we did not run these. Silently dropping to running against prod could introduce confusion if we misconfigure them.

Reply:
Sure will do